tutorials/polkadot-sdk/parachains/connect-to-relay-chain/prepare-parachain.md

```diff
@@ -108,50 +108,37 @@ To modify the default chain specification, follow these steps:
 1. Generate the plain text chain specification for the parachain template node by running the following command
 
     ```bash
-    ./target/release/parachain-template-node build-spec \
-      --disable-default-bootnode > plain-parachain-chainspec.json
+    chain-spec-builder create --relay-chain paseo \
+        --para-id 4508 \
```

Comment:
Should we have a placeholder instead of 4508 here?

Reply:
I think the only reason it's that specific ID is because earlier the para id reserved is `4508`